engine_bs/packages/package.json

```diff
@@ -12,6 +12,7 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
+    "bs-abstract": "^0.17.0",
```

Comment:
bs-abstract has namespace set to `true`. I don't think that will be supported anytimes soon. Because bucklescript intentionally hides the namespacing stuff

Reply:
Uh oh, I wonder what this means for things that depend on bs-abstract (like Relude and bs-decode).

Reply:
Ohh damn. Ok we'll have to think of a different way then...

Reply:
Hmm. Because bucklescript intentionally hides the implementation details of the generated namespace modules, I have no way to access them without a lot of hacks and risk losing compatible with bsb.  @ostera faced the same problem with odoc actually

Reply:
That really reduces the utility of tools that are built around Bucklescript. Would @bobzhang be open to accepting a PR that would allow tooling to look into the implementation details?